lib/modules/dosomething/dosomething_fact/dosomething_fact.module

```diff
@@ -77,6 +77,110 @@ function dosomething_fact_get_mutiple_fact_field_vars($node, $field_names) {
 }
 
 /**
+ * Returns data for multiple specified fields.
+ *
+ * @param object $node The loaded node to collect field data from.
+ * @param array $field_names Array of field machine names to collect data from.
+ * @return array|null
+ */
+function dosomething_fact_get_facts_data($node, $field_names) {
+  $entity = NULL;
+  $facts = [];
+  $sources = [];
+  $language = dosomething_global_convert_country_to_language(dosomething_global_get_current_prefix());
```

Comment:
Is `global $language_content` based on just the path prefix or does it take into account user preference, headers, etc?